executor/hash_table.go

```diff
@@ -130,6 +142,7 @@ func (c *hashRowContainer) GetMatchedRowsAndPtrs(probeKey uint64, probeRow chunk
 			return
 		}
 		if !ok {
+			c.stat.collision++
```

Comment:
this `collision` does not mean the hash collision in the build tale. users may get confused about this message.

Reply:
This is exactly the hash collision in the build table.

Reply:
please change the output info to `probe collision`